docs/docs/use-cases/flux.md

```diff
@@ -0,0 +1,186 @@
+---
+comments: true
+---
+
+# Deploying Applications with Flux and Keptn
+
+Flux is a tool that keeps Kubernetes clusters synchronized with sources
+of configuration (such as Git repositories) and automates updates to
+configuration when new code is deployed.
+
+This section shows how to add Keptn to an existing Flux use case that runs
+[pre and post-deployment jobs with Flux](https://fluxcd.io/flux/use-cases/running-jobs/)
+to make it simpler and more straight-forward.
```

Comment:
```suggestion
[pre and post-deployment jobs with Flux](https://fluxcd.io/flux/use-cases/running-jobs/).
Adding Keptn makes it simpler and more straight-forward
to run the Flux pre and post-deployment jobs
and provides added observability out of the box.
```
We need more information about why one needs both Flux pre/post deployment jobs and Keptn pre/post-deployment tasks and what the relationship is between them.  Maybe some examples of the observability information one gets by implementing Keptn?  And perhaps a quick statement about the difference between Flux jobs and Keptn tasks?  No Keptn evaluations here, right?